Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -3357,9 +3357,9 @@ public static partial class PythonOps {
             return DoubleOps.Compare(self, other);
         }
 
-        [Obsolete("Use Bytes(IList<byte>) or Bytes(ReadOnlySpan<byte>) instead.")]
+        [Obsolete("Use Bytes(IList<byte>) instead.")]
         public static Bytes MakeBytes(byte[] bytes) {
-            return new Bytes(bytes.AsSpan());
+            return Bytes.Make(bytes.ToArray());
```

Comment:
I use `ToArray()` here to maintain the semantics of the method as it was before. Although the usage shows that a non-copying construction would be OK, I am uneasy of providing it here as it is a public method. What about calling `Bytes.Make` directly in the expression tree and deleting `PythonOps.MakeBytes`?

Reply:
I believe the expression tree is used when compiling to an assembly. For example, looking at the compiled code of `b"a"` using ILSpy, I see that it ends up as
```C#
PythonOps.MakeBytes(new byte[4] { (byte)97 })
```
I'm guessing the method needs to be public so that it can be called from external DLLs so that probably rules out using `Bytes.Make`.

It seems like in this case it would be beneficial to allow it to construct the Bytes object without an extra array copy. I'm not sure what the solution is... Maybe add a description to the method and `[EditorBrowsable(EditorBrowsableState.Never)]` and if people choose to disregard it then too bad.

Reply:
OK, let's keep using it as a no-copy construction option and pile up any usage discouraging attributes on it.

I did some tests in trying to use the internal method, but the only version that worked was using reflection. Since constructing a `bytes` object from a constant is such a basic operation, I don't like the extra performance hit of the reflection in this case.